[quarkus-main] openapi-java integration tests native build fails due to unresolved method

### Bug description

Quarkus has upgraded Jackson to 2.18.0 & Swagger Core is using some deprecated APIs which got removed.

```
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered unresolved method during parsing: com.fasterxml.jackson.databind.BeanDescription.findJsonValueMethod(). This error is reported at image build time because class io.swagger.v3.core.jackson.ModelResolver is registered for linking at image build time by command line and command line.
Error encountered while parsing io.swagger.v3.core.jackson.ModelResolver.findJsonValueType(ModelResolver.java:1065)
Parsing context:
   at io.swagger.v3.core.jackson.ModelResolver.resolve(ModelResolver.java:410)
   at io.swagger.v3.core.converter.ModelConverterContextImpl.resolve(ModelConverterContextImpl.java:97)
   at io.swagger.v3.core.converter.ModelConverters.readAll(ModelConverters.java:137)
   at io.swagger.v3.core.converter.ModelConverters.readAll(ModelConverters.java:128)
   at org.apache.camel.openapi.RestModelConverters.readClassOpenApi3(RestModelConverters.java:84)
   at org.apache.camel.openapi.RestModelConverters.readClass(RestModelConverters.java:73)
   at org.apache.camel.openapi.RestOpenApiReader.appendModels(RestOpenApiReader.java:1028)
   at org.apache.camel.openapi.RestOpenApiReader.parse(RestOpenApiReader.java:324)
   at org.apache.camel.openapi.RestOpenApiReader.read(RestOpenApiReader.java:209)
   at org.apache.camel.openapi.RestOpenApiSupport.renderResourceListing(RestOpenApiSupport.java:315)
   at org.apache.camel.openapi.RestOpenApiProcessor.doInit(RestOpenApiProcessor.java:75)
   at org.apache.camel.support.service.BaseService.init(BaseService.java:85)
   at org.apache.camel.support.service.BaseService.start(BaseService.java:115)
   at org.apache.camel.support.ScheduledPollConsumer.beforePoll(ScheduledPollConsumer.java:736)
   at root method.(Unknown Source)
```

--- a/extensions-support/swagger/runtime/src/main/java/org/apache/camel/quarkus/support/swagger/runtime/graal/SwaggerSubstitutions.java
+++ b/extensions-support/swagger/runtime/src/main/java/org/apache/camel/quarkus/support/swagger/runtime/graal/SwaggerSubstitutions.java
@@ -16,21 +16,15 @@
  */
 package org.apache.camel.quarkus.support.swagger.runtime.graal;
 
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.lang.reflect.Type;
 import java.util.Calendar;
 import java.util.List;
 
-import com.fasterxml.jackson.databind.BeanDescription;
-import com.fasterxml.jackson.databind.introspect.AnnotatedMember;
 import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.RecomputeFieldValue;
 import com.oracle.svm.core.annotate.RecomputeFieldValue.Kind;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 import io.swagger.parser.OpenAPIParser;
-import io.swagger.v3.core.jackson.ModelResolver;
 import io.swagger.v3.parser.OpenAPIV3Parser;
 import io.swagger.v3.parser.core.extensions.SwaggerParserExtension;
 import io.swagger.v3.parser.core.models.AuthorizationValue;
@@ -108,24 +102,5 @@ final class CalendarBuilderSubstitutions {
     @Substitute
     public Calendar build() {
         throw new UnsupportedOperationException("Calendar::build is not supported");
-    }
-}
-
-// Cuts out references to deprecated & removed Jackson methods
-// TODO: Remove this https://github.com/apache/camel-quarkus/issues/6593
-@TargetClass(ModelResolver.class)
-final class ModelResolverSubstitutions {
-    @Substitute
-    protected Type findJsonValueType(final BeanDescription beanDesc) {
-        try {
-            Method m = BeanDescription.class.getMethod("findJsonValueAccessor");
-            AnnotatedMember jsonValueMember = (AnnotatedMember) m.invoke(beanDesc);
-            if (jsonValueMember != null) {
-                return jsonValueMember.getType();
-            }
-        } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
-            return null;
-        }
-        return null;
     }
 }

